### ISSUE TITLE
[Quest] Plundering the Plunderers - Polly respawms several times

### ISSUE THREAD

**Issue by padomu** [Atlantiss/NetherwingBugtracker#684]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)

**Description**: 
The rogue Quest "Plundering the Plunderers" https://www.wowhead.com/quest=2381/plundering-the-plunderers might have a wrong mechanics implemented.

**Current behaviour**:
You have to go to the ship close to ratchet at 65.0 45.4. You have to lock pick the chest in the ship. Then polly spawms. If you pick the lock several times, polly spawms several times. I think that's wrong. Polly is Skull-Level (that's okay) but if he spawms several times it can be hard (if even possible) to kill them.

**Expected behaviour**: 

He should only spawm once and/or the box should "dissapear" once he spawmed.

Edit: Also only the one who got the ECAC can loot polly - but that might be wanted.

**Server Revision**: 
2060

**Comment by tcberk12** [comment]
https://www.wowhead.com/quest=2381/plundering-the-plunderers

From a wowhead comment on Dec 12, 2006:
> 5. DO NOT loot the chest again, as polly will repsawn and KILL you.

**Comment by Easyfraggy** [comment]
Closing for now, reopen if you find evidence to the contrary!

**Comment by padomu** [comment]
> https://www.wowhead.com/quest=2381/plundering-the-plunderers
> 
> From a wowhead comment on Dec 12, 2006:
> 
> > 1. DO NOT loot the chest again, as polly will repsawn and KILL you.
> 
> this looks intended.

it's not respawming, polly is there for 2,3,4 times. So 4 times polly. Just didnt looke intendend

**Comment by tcberk12** [comment]
> if you are not pulling aggro, you can re-open it til around 105 lockpick skill, it has a 5sec cd, once you are done, you can run out and hope Polly doesnt kill your sorry ass, or just wait for him to despawn.

It sounds like he is saying that if you have someone there to help you (someone to pull aggro/kill Polly), you can repeat this until 105 skill. Or, if you are alone, you can pick, run away and wait for despawn so you can pick again. In both interpretations, Polly is spawned every time the lock is picked.

To me, it sounds like Polly should just despawn faster?

**Comment by padomu** [comment]
hmm, so I lock picked the chest 2 times in a row.  The respawm timer of the chest is very low. I did that because I didn't see Polly spawming (my bad). So it was there 2 times. I intepret the Quote above as "I lock picked until skill 105, then I had to get out of the ship because I didn't have had the Quest for it and thus I couldn't weaken Polly. (Polly 2hits you without the weakener from the Quest). Thus I just run out there".
